src/utils/streams/index.ts

```diff
@@ -31,6 +35,16 @@ const decodeStream = async (stream: ReadableStream) => {
     }
 
     result.push(...lines);
+
+    if (lineSizeLimit) {
+      if (result[result.length - 1].length > lineSizeLimit) {
```

Comment:
Same comment here about this flaky condition. It doesn't actually iterate over every line (at least in [this log](https://parsley.mongodb.com/evergreen/spruce_ubuntu1604_test_2c9056df66d42fb1908d52eed096750a91f1f089_22_03_02_16_45_12/0/task?bookmarks=0,297) that I'm looking at). You can also see that the lines in the screenshot have not been trimmed to 10 bytes.

<img width="1296" alt="image" src="https://github.com/evergreen-ci/parsley/assets/9298431/c2adbde1-0d96-411c-89aa-8e9298be419e">

Reply:
oof sorry missed this when recreating the pr.

Reply:
Thanks for catching this. I just updated the pr